datasets/Van_de_Schoot_PTSD/convert_clean_schoot_lgmm_ptsd.ipynb

```diff
@@ -0,0 +1,282 @@
+    "\n",
+    "# 62 with both missing titles and abstracts\n",
+    "print(\"Number of papers with missing title AND abstract: \", (all_title.isna() & all_data['abstract'].isna()).sum(), \"\\n\")\n",
+    "\n",
+    "# 764 with either missing titles or abstracts\n",
+    "print(\"Number of papers with missing title OR abstract:  \", (all_title.isna() | all_data['abstract'].isna()).sum(), \"\\n\\n\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total remaining papers in training set:      5782 \n",
+      "\n",
+      "Total number of INCLUSIONS:                  38  ( 0.66 % )\n",
+      "\n",
+      "Total number of EXCLUSIONS:                  5744 \n",
+      "\n",
+      "\n",
+      "Total EXCLUSIONS after abstract screening:   5426 \n",
+      "\n",
+      "Total EXCLUSIONS after full text screening:  318 \n",
+      "\n",
+      "Total INCLUSIONS after full text screening:  30 \n",
+      "\n",
+      "Total INCLUSIONS after abstract screening:   8 \n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# remove duplicates based on just titles\n",
+    "unique_train_data = train_data[(~all_title.str.lower().duplicated()) | all_title.str.lower().isnull()]\n",
+    "\n",
+    "n_train = len(unique_train_data['included'])\n",
+    "n_inc = unique_train_data['included'].sum()\n",
+    "n_exc = n_train-n_inc\n",
+    "\n",
+    "n_aas_exc = (unique_train_data['inclusion_code'] == 0).sum()\n",
+    "n_aft_exc = (unique_train_data['inclusion_code'] == CODE_AFT_EXCLUDE).sum()\n",
+    "n_aft_inc = (unique_train_data['inclusion_code'] == CODE_AFT_INCLUDE).sum()\n",
+    "n_aas_inc = (unique_train_data['inclusion_code'] == CODE_AAS_INCLUDE).sum()\n",
+    "\n",
+    "print(\"Total remaining papers in training set:     \", n_train, \"\\n\")\n",
+    "print(\"Total number of INCLUSIONS:                 \", n_inc, \" (\", round(100*n_inc/n_exc, 2), \"% )\\n\")\n",
+    "print(\"Total number of EXCLUSIONS:                 \", n_exc, \"\\n\\n\")\n",
+    "print(\"Total EXCLUSIONS after abstract screening:  \", n_aas_exc, \"\\n\")\n",
+    "print(\"Total EXCLUSIONS after full text screening: \", n_aft_exc, \"\\n\")\n",
+    "print(\"Total INCLUSIONS after full text screening: \", n_aft_inc, \"\\n\")\n",
+    "print(\"Total INCLUSIONS after abstract screening:  \", n_aas_inc, \"\\n\\n\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "unique_train_data.to_csv(\"PTSD_VandeSchoot_18.csv\", index=False)\n",
+    "os.rename(os.path.join(\"/automated-systematic-review-datasets/datasets/Van_de_Schoot_PTSD/csv\", \"PTSD_VandeSchoot_18.csv\"),\n",
+    "          os.path.join(\"/automated-systematic-review-datasets/datasets/Van_de_Schoot_PTSD/output\", \"PTSD_VandeSchoot_18.csv\"))"
```

Comment:
These 3 lines confuse me a bit, isn't 
`unique_train_data.to_csv("csv/PTSD_VandeSchoot_18.csv", index=False)`
sufficient?

Reply:
Yes, it is sufficient indeed. The other 2 lines just move the output file to a new directory called 'output'. Since I simply converted the original R code into Python, I haven't made any change yet. I'll take the initiative to improve the code.